### DISCUSSION TITLE
Output currently playing to text file

### DISCUSSION THREAD

**Discussion by NotAkitake** [Taiko2k/Tauon#1364] [Ideas]
Mostly for streaming purposes, but having the ability to output currently playing info to a text file to then be picked up by OBS or otherwise would really be handy!

**Comment by Taiko2k** [comment]
Are you on Linux or Windows? On Linux MPRIS already exists to broadcast currently playing info to the system. If you write a script that pulls from that then it will work with any player. (Check out the cli app `playerctl`.)

**Comment by NotAkitake** [comment]
I personally am on Linux and am aware of that solution. However Windows users don´t have that option.
A friend of mine just found out about Tauon and loves it!
He uses both Windows and Linux, and was asking about this feature so I proposed it.

**Comment by C0rn3j** [comment]
I think you could abuse the Listen Along feature for this.

**Comment by Taiko2k** [comment]
There is SMTC on Windows which is the Windows equivalent of MPRIS. Its a little more obtuse to develop for, this post I found goes over it https://stackoverflow.com/questions/65011660/how-can-i-get-the-title-of-the-currently-playing-media-in-windows-10-with-python
